src/Polly.Core/Simmy/Latency/LatencyStrategyOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Polly.Simmy.Latency;
+
+#pragma warning disable CS8618 // Required members are not initialized in constructor since this is a DTO, default value is null
+
+/// <summary>
+/// Represents the options for the Latency chaos strategy.
+/// </summary>
+public class LatencyStrategyOptions : MonkeyStrategyOptions
+{
+    internal static readonly TimeSpan DefaultLatency = TimeSpan.FromSeconds(30);
+
+    /// <summary>
+    /// Gets or sets the delegate that's raised when delay occurs.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>.
+    /// </remarks>
+    public Func<OnLatencyArguments, ValueTask>? OnLatency { get; set; }
+
+    /// <summary>
+    /// Gets or sets the latency generator that generates the delay for a given execution.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>. Either <see cref="Latency"/> or this property is required.
+    /// When this property is <see langword="null"/> the <see cref="Latency"/> is used.
+    /// </remarks>
+    public Func<LatencyGeneratorArguments, ValueTask<TimeSpan>>? LatencyGenerator { get; set; }
+
+    /// <summary>
+    /// Gets or sets the delay for a given execution.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to 30 seconds. Either <see cref="LatencyGenerator"/> or this property is required.
+    /// </remarks>
+    public TimeSpan Latency { get; set; } = LatencyConstants.DefaultLatency;
```

Comment:
Do we validate somewhere that this value is non-negative (allowing for a special-case Infinite value if that's supported)?

Reply:
Great point! I'm doing the same as Timeout strategy, just calling the user delegate.